assets/_scss/components/_search.scss

```diff
@@ -0,0 +1,98 @@
+@mixin usa-search-submit-small {
+  background-image: url('../img/search.png');  
+  background-image: url('../img/search.svg');  
+  background-position: 50%;
+  background-repeat: no-repeat;
+  width: 4.5rem;
+}
+
+@mixin usa-search-submit-default {
+  @include media($mobile-up) {
+    background-image: none;
+    width: 8.5rem;
+  }
+
+  .usa-search-submit-text {
+    display: none;
+
+    @include media($mobile-up) {
+      display: block;
+    }
+  }
+}
+
+input[type="search"] {
+  border: {
+    bottom-right-radius: 0;
+    right: none;    
+    top-right-radius: 0;    
+  }
+  float: left;
+  height: 3.3rem;
+  width: calc(100% - 4.5rem);
+}
+
+.usa-search {
+  max-width: none;
+}
+
+.usa-search_bar {
+  position: relative;
+}
+
+button[type="submit"].usa-search-submit {
+  @include usa-search-submit-small;
+  @include usa-search-submit-default;
+  border: {
+    bottom-left-radius: 0;
+    top-left-radius: 0;
+  }
+  height: 3.3rem;
+  margin: 1.5rem 0;
+  padding: 0;
+  position: absolute;
+
+}
+
+button[type="submit"].usa-search-submit-big {
+  @include media($tablet-up) {
+    @include usa-search-submit-default;
+  }
+
+  @include media($tablet-up) {
```

Comment:
The width I defined in the second media query is not appearing when added inside the first. I tried putting it before and after the include and no dice.

Basically the large button is not getting its larger width in large screen sizes.